staging/src/kubevirt.io/api/instancetype/v1beta1/types.go

```diff
@@ -303,23 +303,49 @@ type VolumePreferences struct {
 type PreferredCPUTopology string
 
 const (
-
 	// Prefer vCPUs to be exposed as cores to the guest
+	//
+	// Deprecated, use cores instead.
 	PreferCores PreferredCPUTopology = "preferCores"
```

Comment:
https://github.com/kubernetes/kubernetes/blob/fad52aedfcc14061cf20370be061789b4f3d97d9/staging/src/k8s.io/api/core/v1/types.go#L3667

Kubernetes adds Depracated prefix to the api fields without changing the values. Should we follow this pattern here?

cc @EdDev

Reply:
@alaypatel07 ACK done sorry for the delay, would you mind taking another look?